spec/services/rent_service_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+describe RentService do
+  let(:user) { create(:user) }
+  let(:council) { create(:council) }
+  let(:other) { build(:user) }
+  let(:rent) { build(:rent) }
+  let(:admin) { build(:admin) }
+  let(:overbook) { build(:rent) }
+  let(:council_rent) { build(:rent, council: council) }
+  let(:booked_council) { create(:rent, council: council, user: user) }
+
+  describe :make_reservation do
+    it :valid_reservation do
+      RentService.reservation(user,rent).should be_truthy
+    end
+
+    it :invalid_reservation do
+      rent.d_from = nil
+      begin
+        RentService.reservation(user, rent)
+      rescue ActiveRecord::RecordInvalid => error
+        error.message.should include(t('activerecord.errors.messages.blank', attribute: Rent.human_attribute_name(:d_from)))
```

Comment:
@hnykda Here it is 100 characters, it depends on the config in `.hound.yml` which points to `.rubocop`.